app/views/grading_reviews/index.html.erb

```diff
@@ -32,6 +34,8 @@
           <td data-sort='<%= term_registration.account.surname %>'><%= highlight_search(term_registration.account.surname, params[:q]) %></td>
           <td data-sort='<%= term_registration.account.matriculation_number %>'><%= highlight_search(term_registration.account.matriculation_number, params[:q]) %></td>
           <td><%= tutorial_group_title(term_registration.tutorial_group) %></td>
+          <td data-sort='<%= term_registration.account.email %>'><%= highlight_search(term_registration.account.email, params[:q]) %></td>
+          <td><%= term_registration.student_group.title %></td>
```

Comment:
Please verify that the `student_group` is eager loaded in the controller. We do not want to introduce n+1 queries.